trulens_eval/examples/snowflake_demo/requirements.txt

```diff
@@ -14,6 +14,7 @@ llama-index-embeddings-huggingface==0.2.0
 llama-index-legacy==0.9.48
 litellm==1.38.12
 streamlit==1.35
+snowflake[ml]
```

Comment:
should this be `snowflake-ml-python`?

Reply:
I forget why, but at the time that didn't work and the docs for cortex search:
https://docs.snowflake.com/LIMITEDACCESS/cortex-search/query-cortex-search-service#label-cortex-search-query-syntax-python
Further linked to this:
https://docs.snowflake.com/developer-guide/snowflake-python-api/snowflake-python-installing
Which said to do `pip install snowflake[ml] -U` at the bottom.